paddle/fluid/platform/cuda_enforce.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "paddle/fluid/platform/enforce.h"
+#ifdef PADDLE_WITH_CUDA
+#ifdef PADDLE_CUDA_KERNEL_CHECK
+
+#include <cuda.h>
+#include <stdio.h>
+
+#define __LEN_ERROR_MSG 128
+
+#define PADDLE_ENFORCE_CHECK_CUDA_KERNEL()                               \
```

Comment:
It is recommended to explain the usage and limit of the new macro, avoid misuse

Reply:
ref: https://github.com/PaddlePaddle/Paddle/blob/7cb4a8b8f21b3432ceeacdc23ddf8b6841cf0e50/paddle/fluid/platform/enforce.h#L436

Reply:
done, thx.